src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Scaffold.java

```diff
@@ -306,7 +306,7 @@ private void fastTower(boolean down, BlockPos checkBlock) {
                     mc.player.setVelocity(0, -0.28f, 0);
             }
         } else {
-            if (mc.options.jumpKey.isPressed() && !mc.options.sneakKey.isPressed() && fastTower.get()) {
+            if (mc.options.jumpKey.isPressed() && !mc.options.sneakKey.isPressed() && fastTower.get() && InvUtils.testInHotbar(stack -> stack.getItem() instanceof BlockItem)) {
```

Comment:
Suppose I have only sand or any unselected block in my hotbar, wouldn't it still start flying in air?

Reply:
okay i changed it to use the validItem method instead of checking if its an instance of BlockItem

Reply:
balls